src/stores/db-migrate.js

```diff
@@ -0,0 +1,22 @@
+import Umzug from 'umzug';
+import path from 'path';
+
+export async function migrate(mongoClient) {
```

Comment:
Small thing, but `runMigrationsScripts` would be a clearer name as to what is migrated from where.

Reply:
Ok, will do